src/sql/query_util.py

```diff
@@ -0,0 +1,10 @@
+from sqlglot import parse_one, exp
+from sqlglot.errors import ParseError
+
+
+def extract_tables_from_query(query):
+    try:
+        tables = [table.name for table in parse_one(query).find_all(exp.Table)]
+        return tables
+    except ParseError:
+        return []
```

Comment:
It means sqlglot was not able to detect any tables in the query, can happen if there are syntax issues

Reply:
So if the table exists and there is a syntax error we return an empty array? 
I would expect `extract_tables_from_query` to throw an error if there is a syntax issue but if that's not possible I think we should document it.

Reply:
This is an example of query where table exists (as stored snippet) but there's a syntax error in the query. It'll throw error:

<img width="1103" alt="Screenshot 2023-05-29 at 12 56 54 PM" src="https://github.com/ploomber/jupysql/assets/29096135/f10bfb9e-e46c-4228-becf-ad84f06bdaab">

@yafimvo

Reply:
Great. 

Since we assume the query is valid, I think we should add a note that this function (`extract_tables_from_query`) doesn't throw errors if the query is invalid.

If someone will want to use this function in the future they should know that if it returns `[]` - it doesn't mean there is no such table.

example:

```
query_util.extract_tables_from_query("select * from table_name limi 2")
```

The table exists but it will return `[]`

P.S
Another option is to check if the query is valid inside `extract_tables_from_query` and throw an error.

Reply:
Currently if I try to raise an exception some fixtures are failing. Added some context in [this issue](https://github.com/ploomber/jupysql/issues/546).
For now, I'll add some comments

@yafimvo